Marsiple/App/Posts/PostsViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  PostsViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 16/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class PostsViewController: UIViewController {
```

Comment:
We have a `retain cycle` within this class file. As an exercise, find it and resolve it.